kafka-rest/src/test/java/io/confluent/kafkarest/unit/AbstractConsumerResourceTest.java

```diff
@@ -34,7 +35,6 @@
 import io.confluent.kafkarest.KafkaRestApplication;
 import io.confluent.kafkarest.KafkaRestConfig;
 import io.confluent.kafkarest.KafkaRestContext;
-import io.confluent.kafkarest.Errors;
```

Comment:
Could we add a simple test to ensure these changes work with the consumer path?

Reply:
Added a integration test for consumer.